variants/Generic/variant.h

```diff
@@ -99,6 +99,11 @@ static const uint8_t SCK  = PIN_SPI_SCK ;
 static const uint8_t SDA = PIN_WIRE_SDA;
 static const uint8_t SCL = PIN_WIRE_SCL;
 
+/*
+ * Reset Button at P0.21
+ */
+#define RESET_PIN            21
```

Comment:
@bojh the Travis CI build is failing, for nRF51,

I suggest we either:

1) Remove it for the generic variant, people might want to use the pin as a GPIO

or

2) Wrap it in an `#ifdef NRF52` for it's only possible for nRF52 boards.

I prefer option 1) but could be convinced about 2) :)

@dlabun any feedback on this?

Reply:
@sandeepmistry May option 1 is more common, So I will remove it for the generic variant.